dex/testing/eth/create-node.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/sh
```

Comment:
Please update shebang and the new-window + new-session commands as per https://github.com/decred/dcrdex/pull/971

Reply:
Looks ok?

Reply:
Need it in the heredocs too. e.g. https://github.com/decred/dcrdex/pull/956/commits/d76d92deab2fba32cfe6883b30ff8438f7f4490e#diff-6fd0fb5ffa4a8526221349744d156b536c50c9959606a486034aa7b807960c61R33

Reply:
ah